core/lib/via_btc_client/src/inscriber/mod.rs

```diff
@@ -98,6 +98,7 @@ impl Inscriber {
         &mut self,
         input: &InscriptionMessage,
         config: InscriptionConfig,
+        recipient: Option<Recipient>,
```

Comment:
What about if we move this inside the `InscriptionConfig` struct?

Reply:
We could, but since it's a BTC transfer that means it's not actually related to the inscription itself...